Contest_API.md

```diff
@@ -1226,6 +1244,70 @@ Returned data:
 {"id":"nicky","type":"admin"}
 ```
 
+### Account
+
+Provides information on which account the client represents within a given contest, which endpoints properties are visible, and what [capabilities](#capabilities) (typically PUT, POST, PATCH, or DELETE actions) this account has access to or can perform.
```

Comment:
I think we still allow public access to the API (i.e. without an account). Maybe we should specify what happens in that case?

Reply:
I was about to just say that it returns 401 when not logged in, but even a public client might be helped by knowing which types and capabilities they have access to. What do you think about making the id and username not required and nullable for this case? Any other suggestion?

Reply:
I was also leaning to making these fields nullable. Let's discuss tonight?

Reply:
k